src/actions/rate.js

```diff
@@ -0,0 +1,55 @@
+import request from 'superagent-es6-promise'
+
+export const UPDATING_PENDING = "UPDATING_PENDING";
+export const RATE_UPDATED_SUCCESS = "RATE_UPDATED_SUCCESS";
+export const RATE_UPDATED_ERROR = "RATE_UPDATED_ERROR";
+
+export const ADDING_PENDING = "ADDING_PENDING";
+export const RATE_ADDED_SUCCESS = "RATE_ADDED_SUCCESS";
+export const RATE_ADDED_ERROR = "RATE_ADDED_ERROR";
+
+export function addRate(rate) {
+    return (dispatch) => {
+        dispatch({type: ADDING_PENDING});
```

Comment:
For logging purposes, add the rate in the action. That way you can see what is being sent